ios/NotificationsHelperIOS10.swift

```diff
@@ -59,11 +60,14 @@ class NotificationsHelperIOS10: NSObject, UNUserNotificationCenterDelegate {
     }))
   }
   
-  private func showRichNotification(title: String, subtitle: String, body: String, url: String) {
+  private func showRichNotification(title: String, subtitle: String, body: String, image: String?, url: String) {
     let content = UNMutableNotificationContent()
     content.title = title
     content.subtitle = subtitle
     content.body = body
+    if let image: String = image! {
```

Comment:
There is a huge diference, the first image might not be a string, and canb e undefined.
The second one is for sure a string as we are using the pattern `if let`, so if we check if it's nil, we still need to unwrap (add the !) when we use it. In the current way we are sure what when we use it will be for sure a string.